src/NSwag.AspNetCore/SwaggerUiSettingsBase.cs

```diff
@@ -35,9 +35,7 @@ public abstract class SwaggerUiSettingsBase : SwaggerSettings
         public SwaggerUiSettingsBase()
         {
             TransformToExternalPath = (internalUiRoute, request) =>
-            {
-                return request.GetBasePath() + internalUiRoute;
-            };
+                '/' + (request.GetBasePath()?.TrimEnd('/') + '/' + internalUiRoute?.TrimStart('/')).Trim('/');
```

Comment:
I think the returned route must start with /, no?

`request.GetBasePath()?.TrimEnd('/') + '/' + internalUiRoute?.TrimStart('/')` should be enough